README.md

```diff
@@ -1 +1,79 @@
+- Scalable architecture
+- Product agnostic approach
+
+## Available APIs
+
+This repository uses Compositions for AWS, Azure, and GCP provider APIs, as well as the Upbound Control Plane provider. For more information, review the provider Configurations below:
+
+- [`Cluster.aws.caas.upbound.io`](https://marketplace.upbound.io/configurations/upbound/configuration-caas/v0.1.0/resources/aws.caas.upbound.io/XCluster/v1alpha1) 
+    - Provision/Manage an EKS Cluster
+
+- [`Cluster.azure.caas.upbound.io`](https://marketplace.upbound.io/configurations/upbound/configuration-caas/v0.1.0/resources/azure.caas.upbound.io/XCluster/v1alpha1) 
+
+    - Provision/Manage an AKS Cluster
+    
+- [`Cluster.gcp.caas.upbound.io`](https://marketplace.upbound.io/configurations/upbound/configuration-caas/v0.1.0/resources/gcp.caas.upbound.io/XCluster/v1alpha1) 
+    - Provision/Manage a GKE Cluster
+
+- [`ControlPlane.mcp.caas.upbound.io`](https://marketplace.upbound.io/configurations/upbound/configuration-caas/v0.1.0/resources/mcp.caas.upbound.io/XControlPlane/v1alpha1)  
+    - Provision/Manage an Upbound Control Plane
+
+- [`Connector.mcp.caas.upbound.io`](https://marketplace.upbound.io/configurations/upbound/configuration-caas/v0.1.0/resources/mcp.caas.upbound.io/XCluster/v1alpha1)
+    - Provision/Manage an MCP Connector
+
+Each API in the [`apis`](https://github.com/upbound/configuration-caas/tree/main/apis) folder contains compositions and definitions for each cloud provider. Once you clone the CaaS repository, you can modify these compositions to fit your organizations needs. 
+    
+## Deployment
+
+To deploy CaaS in a new organization, follow the [Get Started guide](https://docs.upbound.io/quickstart/multicloud-deploy/).
+
+For deployments to an existing organization, log in to your Upbound organization
+and go to **Configurations** and click **Create Configuration**.
+
+Connect your Upbound organization to GitHub. Select the Github organization and
+repository name for your cloned repo and choose the **Cluster as a
+Service** configuration.
+
+When your new configuration is ready, create a new control plane based on the
+cloned repo you just created.
+
+After a few minutes, your new control plane is ready!
+
+To authenticate and configure your providers, see the [multicloud deployment
+guide](https://docs.upbound.io/quickstart/multicloud-deploy/#configure-provider-upbound).
+
+Once authenticated and configured, your self-service Upbound console lists
+available cloud resources.
+
+## GitOps integration
+
+CaaS deployments work best when managed in your infrastructure as code
+lifecycle. 
+
+For more information on how to integrate Argo CD and Flux in your Upbound
+environment, check out the [GitOps with Control Planes doc](https://docs.upbound.io/quickstart/multicloud-deploy/#configure-provider-upbound).
+
+The Upbound Marketplace hosts a [community-owned Argo CD provider](https://github.com/crossplane-contrib/provider-argocd) to manage and
```

Comment:
I think this (he provider-argocd) is sending users down the wrong path vis-a-vis GitOps in relation to APIs on their managed control planes. I think it would be better to point them at this: https://docs.upbound.io/concepts/mcp/control-plane-connector/